lib/cn/omf.h

```diff
@@ -190,14 +193,16 @@ struct kblock_hdr_omf {
     /* metrics */
     uint32_t kbh_entries;
     uint32_t kbh_tombs;
+    uint32_t kbh_rsvd1;
     uint32_t kbh_key_bytes;
-    uint32_t kbh_val_bytes;
+    uint64_t kbh_val_bytes;
```

Comment:
Nice catch!

Reply:
I think it's been broken for a long time, making cn_metrics.py output bogus.  I don't know if there is any actual code internal to hse that relies on this info, you'd think tree maintenance might need accurate data...